### PR TITLE
added upload functionality on admin side

### PR DESCRIPTION
**Pull request by laxmikandel97**

### CODE REVIEW

File: model/Database.php [hunk 1]
```diff
@@ -992,5 +992,42 @@ function checkAdmin($user_id)
         return $statement->fetch(PDO::FETCH_ASSOC);
     }
 
+    /**
+     * update the database to store the path of attachments
+     * @param String $filePath name of the file/attachments
+     * @param int $itemId id of the item to where file paths are going to be stored
+     * @author Laxmi
+     */
+    function updateAttachments($filePath, $itemId)
```
Comment (dsloan92) [inline model/Database.php]: Could this function name be specific to the item table? Oleg also made a similar function but it updates the itemTutorYear table

**Review by dsloan92 — COMMENTED** [review]
Looks good! Just added one quick comment. 
Not really sure what the software key for VS Enterprise document is... but it looks like it's being deleted, so should be good.

Let me know when you make the quick change and I'll merge.

File: model/Database.php (hunk 1 above)
Comment (laxmikandel97) [reply model/Database.php]: sure I will make a quick change. Thank you

**Review by dsloan92 — APPROVED** [review]
Thanks for updating!

**Pull request merged**